apps/frontend/src/screens/Game.tsx

```diff
@@ -95,6 +101,9 @@ export const Game = () => {
           setResult(message.payload.result);
           break;
 
+        case OPPONENT_DISCONNECTED:
+          setResult(OPPONENT_DISCONNECTED);
```

Comment:
Is this needed?

Reply:
oops got it duplicated while merging, removed duplicate